rock_paper_scissors.py

```diff
@@ -150,8 +150,8 @@ def main():
     game.rounds = int(input('How many odd rounds would you like to play? '))
 
     while not game.is_complete:
-        user.choice = input('Choose rock, paper, or scissors: ')
-        computer.choice = random.choice(VALID_CHOICES)
+        user.choice = input('Choose from the following: {}: '.format(', '.join(game.valid_choices)))
```

Comment:
I have to ask... difficulty of implementations being equal, which prompt do _you_ prefer?
1. `'Choose rock, paper, or scissors: '`
2. `'Choose from the following: rock, paper, or scissors: '`